documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -259,6 +225,30 @@ The connector uses the logical name in a number of ways: as the prefix for all t
 You should give each MongoDB connector a unique logical name that meaningfully describes the source MongoDB system.
 We recommend logical names begin with an alphabetic or underscore character, and remaining characters that are alphanumeric or underscore.
 
+// Type: concept
+// Title: How {prodname} MongoDB connectors perform offset consolidation
+// ModuleID: how-debezium-mongodb-connectors-perform-offset-consolidation
+[[mongodb-offset-consolidation]]
+=== Offset consolidation
+The {prodname} MongoDB connector no longer supports `replica_set` connections to sharded MongoDB deployments. 
+As a result, the offsets recorded by connector versions that used the `replica_set` connection mode are incompatible with the current version.
+
+To minimize the way that the connection mode change affects the connector, and to prevent the connector from running unnecessary snapshots, after you upgrade the {prodname} MongoDB connector, it runs an offset consolidation procedure.
```

Comment:
```suggestion
To minimize the affects of the connection mode change, and to prevent the connector from running unnecessary snapshots, when the connector restarts after the upgrade, it runs a procedure to consolidate offsets.
```